portfolio/src/main/webapp/js/projects.js

```diff
@@ -98,10 +92,12 @@ class Project {
      * @param {string} description
      * @param {string[]} tags
      */
-    constructor(name, description, tags){
+    constructor(id, name, description, tags){
```

Comment:
Don't forget to add this to the documentation.

Reply:
Done.